packages/components-js/tests/e2e/puppeteer/specs/table.e2e.ts

```diff
@@ -208,9 +157,11 @@ describe('lifecycle', () => {
 
     const status = await getLifecycleStatus(page);
 
-    expect(status.componentDidLoad.all, 'final componentDidLoad: all').toBe(32);
-    expect(status.componentDidLoad['p-icon'], 'final componentDidLoad: p-icon').toBe(5);
-    expect(status.componentDidUpdate.all, 'final componentDidUpdate: all').toBe(5);
+    // after adding sorting to every column (5 columns), we get 5 p-buttons-pure and 5 p-icons extra, so that the component amount increases from 30 to 40
```

Comment:
Comments are a bit useless or even confusing outside of the PR.

Reply:
@denyo 
`the component amount increases from 30 to 40` - this comment is not about this PR, but it means "after adding sorting inside of this test (in line 147 it's without sorting) - the components' amount increases". So it increases not because of changes in this PR, but because of the test scenario.

I don't think this comment is useless, because it's not clear without comment, why after adding sorting the components' amout increases.

Generally for me it wasn't  at the start clear, how we calculate the components' amount and how do we get these numbers. So I think for people like me any additional comment might be useful in the future.